gnssanalysis/gn_io/sp3.py

```diff
@@ -59,12 +54,10 @@ def sp3_pos_nodata_to_nan(
         & (sp3_df[("EST", "Y")] == SP3_POS_NODATA_NUMERIC)
         & (sp3_df[("EST", "Z")] == SP3_POS_NODATA_NUMERIC)
     )
-    sp3_df.loc[nan_mask, [("EST", "X"), ("EST", "Y"), ("EST", "Z")]] = _np.NAN
```

Comment:
As I understand `np.nan` `np.NaN` `np.NAN` are all equivalent? If that's the case, we'll just need to choose one and be consistent. Happy to go with `_np.nan` unless there are clear objections from others / an argument to be made that this choice is detrimental somehow